RPN_Tests/InitializeTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Net;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RPN_Calculator;
+using RPN_Database;
+using RPN_TcpServer;
+
+namespace RPN_Tests
+{
+    [TestClass]
+    public class InitializeTests
+    {
+        [TestMethod]
+        public void IPAdressCheckTest()
+        {
+            try
+            {
+                var port = 1024;
+                var ipAddress = IPAddress.Parse("0.0");
+                ResponseServer<double> rpnServer = new ResponseServerAsync(ipAddress, port, RPNCalculator.Calculate, Encoding.ASCII, ContextBuilder.CreateRpnContext);
+                Assert.Fail();
```

Comment:
Z tego co mi wiadomo to jest metoda Assert.Throws, która służy właśnie do testowania rzucania wyjątków: "Assert.Throws returns the exception that's thrown which lets you assert on the exception."
```
var ex = Assert.Throws<Exception>(() => user.MakeUserActive());
Assert.That(ex.Message, Is.EqualTo("Actual exception message"));
```
Link: https://stackoverflow.com/questions/1609536/how-do-i-use-assert-throws-to-assert-the-type-of-the-exception/20365751